addon/components/image-layer.js

```diff
@@ -1,19 +1,43 @@
+import Ember from 'ember';
 import BaseLayer from 'ember-leaflet/components/base-layer';
 
+const {
+  deprecate,
+  isPresent
+} = Ember;
+
 export default BaseLayer.extend({
 
   leafletRequiredOptions: [
-    'imageUrl', 'bounds'
+    'url', 'bounds'
   ],
 
   leafletOptions: [
     'attribution', 'opacity'
   ],
 
   leafletProperties: [
-    'url', 'opacity'
+    'url', 'opacity', 'bounds'
   ],
 
+  init() {
+    const imageUrl = this.get('imageUrl');
```

Comment:
> Sorry for being pedantic, but I believe this kind of consistency makes the project easier to read and gives some extra credibility.

I share your opinion.

Our linter is configured to force `const`, if the value is not changed. Generally we strive for immutability.

Personally I disagree with `ember-suave/no-const-outside-module-scope`, but of course I'll change it. :wink: